src/main/resources/alma/fix/relatedRessourcesAndLinks.fix

```diff
@@ -149,6 +149,25 @@ do list(path: "830??", "var": "$i")
   end
 end
 
+# containedIn
+# TODO: What is the difference between containedIn and isPartOf?
```

Comment:
Should this question be solved right now or do we want to postpone it ?

Reply:
Found some documentation on that: https://github.com/hbz/lobid-resources/issues/1400#issue-1347591143

`containedIn` is used with articles that are ressources depending on an umbrella publication as an journal issue or an anthology. The ressource is not published by itself.

`isPartOf`: is used with ressources that are published by themself but are part of an series, journal etc.

https://lobid.org/resources/api